lms/static/sass/base/_base.scss

```diff
@@ -130,7 +130,6 @@ a:focus {
   margin: 0 auto 0;
   padding: ($baseline*2) 0;
   max-width: grid-width(12);
-  min-width: 760px;
```

Comment:
@marjev, this may be related to the breakpoints and responsive scale you've altered elsewhere, but can you confirm:

1. Why you removed this value?
2. That you've visually proofed that this min-width isn't needed elsewhere in the platform (Courseware, Discussions, Login, etc.)?